src/General/ProfilePicture/ProfilePicture.tsx

```diff
@@ -9,62 +9,51 @@ import {
   ProfilePictureContent,
 } from '../../Style/General/ProfilePictureStyle';
 
-class ProfilePicture extends React.Component<Props, State> {
-  state = {
-    isHover: false,
-  };
-
-  handleMouseEnter = () => {
-    const { editable } = this.props;
-
+const ProfilePicture = ({
+  editable,
+  children,
+  className,
+  ...defaultProps
+}: Props) => {
+  const [isHover, setIsHover] = React.useState<boolean>(false);
+
+  const handleMouseEnter = React.useCallback(() => {
     if (editable) {
-      this.setState({ isHover: true });
+      setIsHover(true);
     }
-  };
-
-  handleMouseLeave = () => {
-    const { editable } = this.props;
+  }, [editable, setIsHover]);
 
+  const handleMouseLeave = React.useCallback(() => {
```

Comment:
same as https://github.com/glints-dev/glints-aries/pull/333#discussion_r424148410

Reply:
Changed to regular function.